### DISCUSSION TITLE
Dataset edit permission grant at role level instead of user level

### DISCUSSION THREAD

**Discussion by crazyisjen** [apache/superset#26367] [Q&A / Help]
I am one of the data team member to import the dataset structure in superset, however everytime when I created the dataset, my team member couldn't edit it by default, is there a way to add the role name in permission field (not the owner name) in case there are someone left company or someone new join company, easier for me to maintain the role list.

I saw one of the item below seems resolve but not right to enable it.
**can override role permissions on Superset**

many thanks !!

**Comment by crazyisjen** [comment]
push

**Comment by Jacobihi** [comment]
Our organization would also love to see role-based ownership for datasets for the administrative benefits that roles provide for access.

**Comment by xenago** [comment]
This is a needed feature. Every aspect of superset should allow role rather than user-level permissions. Single-user level permissions aren't scalable.

**Comment by cesar-loadsmart** [comment]
Another possible solution would be using the tagging system to allow users to edit both charts and datasets with that specific tag. Please make this happen.

**Comment by doughss** [comment]
We're also facing a similar issue in our team. In cases where a teammate leaves and he's the only owner, it becomes impossible to manage permissions until an administrator intervenes.

A feature allowing group or role-based ownership would be very helpful. It would reduce the dependency on individual user ownership and make dataset management much more scalable and resilient.
